### DISCUSSION TITLE
Are there plans to add composables to VueUse?

### DISCUSSION THREAD

**Discussion by shinGangan** [nuxt/scripts#188] [Ideas]
### Context

I believe that the composables available in Nuxt Scripts will be useful not only to Nuxt users, but also to Vue.js users.
Are you planning to repackage them as VueUse? (ex. `vueuse/scripts`)

If this is the direction you are heading in, we would love to help in any way we can.

**Comment by shinGangan** [comment]
Hi @harlan-zw , I was a bit curious so I opened the discussion.
I would like to happy if you could share your opinion when you have time.

**Reply by harlan-zw** [reply]
Hey, thanks for the discussion. Many of these composables may be exported through Unhead as part of the v2.

**Reply by shinGangan** [reply]
@harlan-zw 

Thanks for comments :)

I think it's a great idea to provide the library on Unhead instead of separating it.
If work on Unhead starts, I'll contribute where I can.

**Comment by reslear** [comment]
I agree that would be cool. 

@shinGangan also for now use https://unhead.unjs.io/usage/composables/use-script

**Comment by harlan-zw** [comment]
No plans to mirror these into VueUse — the underlying primitive lives in [`unhead`](https://unhead.unjs.io/usage/composables/use-script), so Vue users can use `useScript` from there directly. Closing 🙏
